### PR TITLE
Issue 101 splash screen fade out effects

### LINKED ISSUE #101: Splash Screen FADE OUT effects
## Basic Information
**Based on the current splash screen add more feature to make the splash screen more smooth**

## Other Information
**Relevant to the #12**

### PR DESCRIPTION
**Pull request by Mingyu-Lian**
## Change Summary
**add splash screen animation: fade in and fade out**
**also the fade in effect for the main components (now is the Pingpong Test), need to be adjusted when the home page done

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

# Related issue

- Resolve #101

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "is-inside-container": "^1.0.0",
     "lucide-react": "^0.395.0",
     "next": "^14.2.4",
-    "next-themes": "^0.3.0",
```
Comment (yunho7687) [inline client/package.json]: don't delete this please

**Review by yunho7687 — CHANGES_REQUESTED** [review]
I will take care about this issue later but for now please check penni channel first

File: client/package.json (hunk 1 above)
Comment (Mingyu-Lian) [reply client/package.json]: I didnt do about this part, might due to the merge.
But anyway, I already add this into the branch and commit again

**Comment by Mingyu-Lian** [comment]
Close as it already transfer to #112

**Pull request closed without merging**